minecraft_servers/Inifnityheroes/manifest.json

```diff
@@ -0,0 +1,45 @@
+{
+  "server_name": "Inifnityheroes",
+  "nice_name": "Inifnityheroes Network",
+  "direct_ip": "play.ifheroes.de",
+  "server_wildcards": [
+    "%.ifheroes.de",
+    "%.inifnityheroes.de"
+  ],
+  "social": {
+    "web": "https://inifnityheroes.de",
+    "web_support": "https://help.inifnityheroes.de",
+    "twitter": "ifheroesserver",
+    "twitch": "",
+    "instagram": "ifheroesserver",
+    "youtube": "https://www.youtube.com/channel/UCavLhm1ZWWMinOcsr6uC_aw",
+    "discord": "https://ifheroes.de/discord"
```

Comment:
Custom links only supported for partners. Please use the discord invite link (so we can guarantee a correct link).